src/components/users/PublicProfile.tsx

```diff
@@ -150,4 +149,7 @@ export const ProfileATag = ({ uid, className = ProfileATagStyle }: ProfileATagPr
 const ProfileATagStyle = 'text-primary font-bold hover:underline'
 
 const ChangeUsernameLink = forOwnerOnly(() =>
-  <Link href='/account/changeUsername'><a className='text-sm link'>Edit</a></Link>)
+  <Link href='/account/changeUsername'>
+    <a className='text-sm inline-flex space-x-2 items-center whitespace-nowrap cursor-pointer disabled:cursor-auto disabled:opacity-50 border rounded-md border-gray-800 text-black drop-shadow-sm hover:ring-1 px-2 py-0.5'>Edit
```

Comment:
One final request and we're good to merge.  Now that the button is next to other buttons, can you change label to 'Edit profile' so that it's less confusing?  Thank you.

<img width="276" alt="Screenshot 2023-07-14 at 8 55 17 AM" src="https://github.com/OpenBeta/open-tacos/assets/3805254/76928c0e-a348-43f0-980e-07ead4b6935e">